src/clarity-angular/data/datagrid/datagrid.spec.ts

```diff
@@ -356,6 +378,26 @@ class TrackByTest {
 
 @Component({
     template: `
+    <clr-datagrid [(clrDgSingleSelected)]="selected">
+        <clr-dg-column>First</clr-dg-column>
+        <clr-dg-column>Second</clr-dg-column>
+    
+        <clr-dg-row *clrDgItems="let item of items;" [clrDgItem]="item">
+            <clr-dg-cell>{{item.value}}</clr-dg-cell>
+            <clr-dg-cell>{{item.value * item.value}}</clr-dg-cell>
+        </clr-dg-row>
+    
+        <clr-dg-footer (click)="selected = null">{{selected?.value}}</clr-dg-footer>
+    </clr-datagrid>
+`
+})
+class SingleSelectionTest {
+    items = [{value: 1}, {value: 2}, {value: 3}];
+    selected: number;
```

Comment:
I know we don't run our unit tests in AoT yet, but this would fail. Your `selected` property receives an object of type `{value: number}`, not an actual `number`.

Anyway, why switch to objects and not use simple numbers like the other tests? Especially since testing object equality in Jasmine opens up a traps that could easily be avoided with native numbers.